### ISSUE TITLE
Input events, select, drag, resize, do not work correctly in Safari (macOS & iPad)

### ISSUE THREAD

**Issue by stahlmanDesign** [BHP-DevHub/react-timeline-9000#153]
I can live with it not working in desktop Safari, but I'd like it to work on iPad.

In Chrome & Firefox I can select items and move/resize them. Very nice feature.

In Safari 12.0.3 I cannot move or resize anything. I can select but after clicking to unselect, I can no longer select anything, even after regenerating the timeline. Input events seem to be handled differently in Safari.

**Comment by vasdee** [comment]
If you are talking about the demo, it needs a bit of love, we haven't updated it in a while 

I can confirm that the library in general does work in Safari though, a dev on our team that uses osx had it working last week. I can dig up some more information on specific versions if you like

**Comment by stahlmanDesign** [comment]
Yes, I was referring to the [demo](http://bhp-react-timeline-9k.s3-website-ap-southeast-2.amazonaws.com)

I also recreated the demo to see if it worked with Create-React-App (it does), by using the latest npm version: `react-timeline-9000` v1.0.12

It works but has the same problems as the demo.

I hope the dev team gets it to work with Safari and can publish a new npm version.

**Comment by vasdee** [comment]
There's only 1 guy on the team that has a mac, so no guarantees on when he'll get around to it :) 

This problem might be related to interactjs, which is something I'd like to move away from in the future. Maybe a version bump of interact in the meantime might fix it?
